modules/client-configuration/pages/autoinst-unattended.adoc

```diff
@@ -0,0 +1,62 @@
+[[autoinst-unattended]]
+= Unattended Provisioning
+
+The "bare metal" functionality allows you to register any new machine as soon as it connects to the local network, using a generic PXE boot image.
+You then go to {productname} {webui} and assign a profile to this machine.
+The next time the client boots, the operating system is installed according to that profile.
+For information about the bare metal provisioning, see xref:client-configuration:autoinst-unattended.adoc#bare-metal[Bare Metal Provisioning].
+
+If you don't want to use the bare metal functionality, you can still declare manually the system at {productname}.
+The {productname} API allows you to create profiles for systems as if they were collected by the bare metal functionality.
+For information about declaring systems with the API, see xref:client-configuration:autoinst-unattended.adoc#create-system-profile[Create a System Profile Manually].
+
+
+[[bare-metal]]
+== Bare Metal Provisioning
+
+When you have the bare metal provisioning option enabled, any bare metal client connected to the {productname} network is automatically added to the organization as soon as it is powered on.
+When this operation is complete, the client is shut down, and it appears in the [guimenu]``Systems`` list, ready to be installed.
+
+
+
+.Procedure: Provisioning Bare Metal Clients
+. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Bare-metal systems].
+. Click btn:[Enable adding to this organization].
+. In the {productname} {webui}, navigate to [guimenu]``Systems``, select the client to provision, and go to the menu:Provisioning[Autoinstallation] tab.
+. Select the {ay} profile to use, and click btn:[Schedule Autoinstallation and Finish] to start the autoinstallation.
+
+New bare metal clients are added to the organization that belongs to the administrator who enabled the bare metal feature.
+To change the organization new clients are added to, disable the bare metal feature, log in as the administrator of the new organization, and then re-enable the feature.
+You can migrate already registered systems to other organizations if required, using the [guilabel]``Migrate`` tab.
+
+You can use the system set manager (SSM) with bare metal clients.
+However, not all SSM features are available for bare metal clients, because they do not yet have an operating system installed.
+This also applies to mixed sets that include bare metal systems.
+All features become available to the set when all the clients in the set have been provisioned.
+For more information on SSM, see xref:client-configuration:system-set-manager.adoc[].
+
+The server uses TFTP to provision the bare metal client, so the appropriate port and networks must be configured correctly in order for provisioning to be successful.
```

Comment:
This is not 100% correct. I would split this into 2 parts:

1. enable bare metal provisioning feature: this should explain how to enable it and talk about the consequences => all systems gets registered to the org where this feature was enabled. This should also talk how to "Transfer" the clients to the final Org.
2. This should talk about changing the Bootstrap entitlled system into a regular one. Assign an autoyast profile and create a cobbler entry. The button "Schedule Autoinstallation and Finish" does not exist in this case !

Reply:
I think I took this button's name from previously existing text without checking. My bad.

Trying to reorganize this, you will tell me if that is better.

Reply:
Definitiv better than before :+1: